package.json

```diff
@@ -9,12 +9,13 @@
   },
   "files": [
     "bin/",
-    "lib/",
+    "src/",
```

Comment:
Sorry about that. I was expecting to merge it on our fork first. My mistake. This change was not supposed to be here.